### ISSUE TITLE
Alt text doesn't appear to be accessible in GraphQL

### ISSUE THREAD

**Issue by tkatchenko** [dennispassway/sanity-plugin-media-library#26]
When I run `*[_type == 'sanity.imageAsset']{alt}` in the Sanity studio Vision tab I get a list of image assets and their alt text but when I run:
```
{
  allSanityImageAsset {
    alt
  }
}
```
in the deployed GraphQL playground, I get a "Cannot query field 'alt' on type 'SanityImageAsset'.".

Is this the correct structure for the getting the alt text? Not sure if I'm missing something as I can't seem to find it via GraphQL.
